### ISSUE TITLE
[Skia/WPF] Uno UI Test support

### ISSUE THREAD

**Issue by francoistanguay** [unoplatform/uno#9215]
### Discussed in https://github.com/unoplatform/uno/discussions/9213

<div type='discussions-op-text'>

<sup>Originally posted by **bstrother1** July  5, 2022</sup>
Am I missing something here? : 
`  <TextBlock AutomationProperties.AutomationId="ATextBlock" Text="Hello World"/>`

but searching for it here:
   `Assert.AreEqual(textLabel , this.AppSession.FindElementByAccessibilityId("ATextBlock").Text);`

throws an Element not found error. 
Is this a bug? To add onto this xaml has:  Property 'AutomationId' is not attachable to elements of type 'TextBlock' when hovering over it. Same things happens on any element with AutomationId attached.</div>
